client/routes/pageNotFound.html

```diff
@@ -1,5 +1,5 @@
 <template name="pageNotFound">
   <h1>{{_ "Error_404"}}</h1>
   <h3>{{_ "Oops_page_not_found"}}</h3>
-  <p>{{_ "Sorry_page_you_requested_does_not_exists_or_was_deleted"}}</p>
```

Comment:
Thank you for address this misspelling, however this is just a translation key. All files under `packages/rocketchat-i18n/i18n` must be updated.

Reply:
I looked at `packages/rocketchat-i18n/i18n/en.i18n.json`. The key value pair corresponding to error is `"Error_404": "Error:404"`, which seems correct. Could you please elaborate what needs to be changed in the `i18n` folder? I'll make the necessary changes.

Reply:
It has nothing to do with `Error_404`, but with `Sorry_page_you_requested_does_not_exists_or_was_deleted` itself. Just look at [the file for English translation](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-i18n/i18n/en.i18n.json#L2605). By updating the key to `Sorry_page_you_requested_does_not_exist_or_was_deleted`, you need to update it in the translation files too, because the text is retrieved from there.

Reply:
I have made the required changes. Please have a look.